### PR TITLE
Add vcf2maf-lite tool

### PR DESCRIPTION
**Pull request by rmadupuri**
- Moved code from https://github.com/genome-nexus/annotation-tools.
- Added `--retain-info` and `--retain-fmt` arguments that allows users to pass custom INFO and FORMAT keys to extract their desired columns - other than the standard MAF columns we extract by default.
- Added new tests and a git actions workflow to trigger tests on push and pull requests.
- Added a Dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim-buster
```
Comment (hweej) [inline Dockerfile]: Python 3.8 EOL is October 24, 2024. Would going to 3.9 or 3.10 be an issue for this script?

File: vcf2maf_lite.py
Comment (hweej) [inline vcf2maf_lite.py:1]: Everything looks good here. Is the spacing doubled? Could we change the indentation to be consistent with the `vcf2maf_lite_tests.py`?

File: Dockerfile (hunk 1 above)
Comment (rmadupuri) [reply Dockerfile]: Updated to 3.10. Thanks Jason!

File: vcf2maf_lite.py
Comment (rmadupuri) [reply vcf2maf_lite.py:1]: Done!

**Pull request merged**